docs/conf.py

```diff
@@ -325,7 +325,9 @@
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://docs.python.org/': None}
+intersphinx_mapping = {
+    'python': ('https://docs.python.org/3', None)
```

Comment:
I did because there is a issue when tries to create documentation. 
See https://travis-ci.org/hachreak/invenio-pages/builds/159235041

```
Warning, treated as error:

/home/travis/build/hachreak/invenio-pages/invenio_pages/models.py:docstring of invenio_pages.models.Page.validate_template_name:None: WARNING: py:exc reference target not found: AssertionError
```

Any idea why?